lib/bootstrap/bundler.rb

```diff
@@ -56,10 +56,19 @@ def setup!(options = {})
     # execute bundle install and capture any $stdout output. any raised exception in the process will be trapped
     # and returned. logs errors to $stdout.
     # @param options [Hash] invoke options with default values, :max_tries => 10, :clean => false, :install => false, :update => false
-    # @param   options[:update] must be either false or a String or an Array of String
+    # @param options[:update] must be either false or a String or an Array of String
+    # Available options are:
+    # * max_tries: The number of times bundler is going to try the installation before failing (default: 10)
+    # * clean: It cleans the unused gems (default: false)
+    # * install: Run the installation of a set of gems defined in a Gemfile (default: false)
+    # * update: Update the current environment (default: false)
+    # * local: Do not attempt to fetch gems remotely and use the gem cache instead (default: false)
+    # * package: Locks and then caches all dependencies to be reused later on (default: false)
+    # * all: It packages dependencies defined with :git or :path (default: false)
```

Comment:
should'nt we default to `:all => true`? it seems like a least-surprise default no?